core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -182,6 +182,7 @@ public static Status httpStatusToGrpcStatus(int httpStatusCode) {
     }
     switch (httpStatusCode) {
       case HttpURLConnection.HTTP_BAD_REQUEST:  // 400
+      case 431: // Request Header Fields Too Large
```

Comment:
A change like this needs to be done in https://github.com/grpc/grpc/blob/master/doc/http-grpc-status-mapping.md first. Why is this change necessary?